contrib/raftexample/raft.go

```diff
@@ -353,6 +340,13 @@ func (rc *raftNode) maybeTriggerSnapshot() {
 		return
 	}
 
+	// wait until all committed entries are applied
```

Comment:
Please add: 
// commitC is synchronous channel, so consumption of the message signals 
// full processing of previous messages.

Reply:
Addressed.